### PR TITLE
Event Loop Monitor + JSDOM Conflict 

### PR DESCRIPTION
**Pull request by leonardoventurini**
- Include and refactor Event Loop Monitor package into the project;
- Make sure the Event Loop Monitor uses the correct `now` function when running in Node.js;
- Implement tests and make sure they are reliable.

### CODE REVIEW

File: lib/event_loop_monitor.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+    setTimeout(function () {
+      let end = self._now();
+      let elapsedTime = end - start;
+      let realDiff = elapsedTime - self.timeoutMillis;
+      let lag = Math.max(0, realDiff);
+
+
+      if (!self._stopped) {
+        self.emit('lag', lag, realDiff);
```
Comment (zodern) [inline lib/event_loop_monitor.js:77]: Is the realDiff used anywhere?

File: tests/event_loop_monitor.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    // Saturate the event loop so that we can detect lag.
+    for ( let i = 0; i < 10000; i++ ) {
+      setTimeout(() => {}, 99);
+    }
+
+    Meteor.setTimeout(function () {
+      const status = monitor.status();
+
+      console.log(process.version);
```
Comment (zodern) [inline tests/event_loop_monitor.js:18]: There are a few console.log's in this file. Are they still needed?

File: lib/event_loop_monitor.js (hunk 1 above)
Comment (leonardoventurini) [reply lib/event_loop_monitor.js:77]: I was going to use it, but I figured out the problem first, I will remove it.

File: tests/event_loop_monitor.js (hunk 2 above)
Comment (leonardoventurini) [reply tests/event_loop_monitor.js:18]: I will do a few more test runs, and once I feel they are completely safe I will remove it.

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -40,11 +43,12 @@ jobs:
         run: |
           curl https://install.meteor.com | /bin/sh
           npm i -g @zodern/mtest
+          meteor npm install
```
Comment (zodern) [inline .github/workflows/test.yml:46]: It looks like this is now unneeded.

Comment (leonardoventurini) [reply .github/workflows/test.yml:46]: Will check

**Pull request merged**